recipes/cad_to_dagmc/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "cad_to_dagmc" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b6b7f34dec0d54a19a9476a2c5c157ef23eb3d6796af86dee6798cdc2420fda9
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py<310]
+  skip: True  # [py>=312]
```

This looks like it could be noarch. Why aren't you using that?
```suggestion
  noarch: python
```

---

Thanks for this. I thought all the dependencies had to be noarch for the recipe to also be noarch. So I noticed [moab](https://github.com/conda-forge/moab-feedstock/blob/main/recipe/meta.yaml) and other dependencies are not noarch and thought I therefore couldn't be.

Glad to hear that I'm wrong there. Keen to make this a noarch recipe.

---

I had a go at moving to noarch but could no longer skip the windows build.

Unfortunately windows builds won't work for now as moab doesn't build a conda package for windows [link to moab meta.yml skip windows line](https://github.com/conda-forge/moab-feedstock/blob/7707e337d01852ee60cfc8baed87501e15ff7ff5/recipe/meta.yaml#L48)

I believe there is a longer term plan to move the moab conda build from autotools to cmake then add windows support. So I shall keep an eye on that and then could come back and have another go at making this package noarch in the future if/when moab builds conda packages for windows.

Perhaps noarch is not necessary for now?